config/core/999-cache.yaml

```diff
@@ -1,25 +0,0 @@
-# Copyright 2018 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: caching.internal.knative.dev/v1alpha1
-kind: Image
-metadata:
```

Comment:
Did you mean to drop this? 🤔

Reply:
Nah, I just did because it annoyed me on reapply locally :joy:. Will be taken out once I have a final PR for this.

Reply:
Was pushed by accident.